### PR TITLE
MMT-3793: Add script to download umm schema files at deployment

### PR DESCRIPTION
**Pull request by htranho**
# Overview

### What is the feature?

Download and process umm schema files at deployment

### What is the Solution?

Create script called by deployment script

### What areas of the application does this impact?

All umm schema files

# Testing

Test all forms of all available types: C, S, Var and T

### Attachments

N/A

# Checklist

- [ ] I have added automated tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by dmistry1 — APPROVED** [review]
Looks good, we should do a test deployment to SIT verifying the script

**Review by cgokey — CHANGES_REQUESTED** [review]
vitest is failing

**Comment by htranho** [comment]
> 

Resolved

File: bin/download_schemas.sh [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    #Reads version number
+    schema_version=`jq '.ummVersions.ummT' ../static.config.json`
+    schema_version=${schema_version//\"/}
+    #Download
+    curl https://git.earthdata.nasa.gov/projects/EMFD/repos/unified-metadata-model/raw/tool/v${schema_version}/umm-t-json-schema.json > umm-t-json-schema-temp.json
+    if [ $? -ne 0 ]; then
+      echo "Failed downloading umm-t-json-schema.json"
+      exit 1
+    fi
+    jq --arg ummJsonSchemaUrl "$ummJsonSchemaUrl" '."$schema" = $ummJsonSchemaUrl' umm-t-json-schema-temp.json > umm-t-json-schema.json
```
Comment (cgokey) [inline bin/download_schemas.sh:75]: I wonder if we can clean up these -temp files when done?

Comment (htranho) [reply bin/download_schemas.sh:75]: Added cleanup to remove temp files

File: bin/download_schemas.sh [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    #Create js file
+    echo "$(echo "const ummTSchema =")" "$(jq '.' umm-t-json-schema.json)" > ummTSchema.js
+    echo "$(echo "export default ummTSchema")" >> ummTSchema.js
+}
+
+cleanup() {
+  rm ./*-temp.*
```
Comment (cgokey) [inline bin/download_schemas.sh:82]: I wonder if you should put a \ in front of it, depending on the shell env, it may prompt "are you sure?"

Comment (cgokey) [reply bin/download_schemas.sh:82]: `\rm ./*-temp.*`

Comment (htranho) [reply bin/download_schemas.sh]: Done

**Review by cgokey — APPROVED** [review]

File: bin/download_schemas.sh (hunk 1 above)
Comment (cgokey) [reply bin/download_schemas.sh:75]: I just ran locally, there is still some files leftover in bin directory can these be cleaned up too?

Comment (htranho) [reply bin/download_schemas.sh:75]: Yes, they all now removed

**Pull request merged**